.github/workflows/template-quality-check.yml

```diff
@@ -0,0 +1,47 @@
+name: quality-check
+
+on:
+  workflow_call:
+    inputs:
+      working_directory:
+        required: true
+        type: string
+
+jobs:
+  check:
+    name: ${{ matrix.os }} with Python ${{ matrix.python-version }}
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os:
+          # TODO: Re-enable when testing done
+          # - windows-latest
```

Comment:
Re-enable before merge.

Reply:
Still going to leave `windows` disabled until `Kaliedo` is resolved. Marked in file